package.json

```diff
@@ -55,7 +55,6 @@
         "@testing-library/jest-dom": "^6.4.6",
         "@testing-library/react": "^16.0.0",
         "@testing-library/user-event": "^14.5.2",
-        "@types/classnames": "^2.3.1",
```

Comment:
pnpm kept telling me that this library ships with its types built-in now... so we don't need this dep.